src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStartedService.ts

```diff
@@ -455,7 +457,9 @@ export class WalkthroughsService extends Disposable implements IWalkthroughsServ
 				id: string;
 			};
 			this.telemetryService.publicLog2<GettingStartedAutoOpenEvent, GettingStartedAutoOpenClassification>('gettingStarted.didAutoOpenWalkthrough', { id: sectionToOpen });
-			this.commandService.executeCommand('workbench.action.openWalkthrough', sectionToOpen);
+			this.commandService.executeCommand('workbench.action.openWalkthrough', sectionToOpen, {
+				inactive: this.layoutService.hasFocus(Parts.EDITOR_PART) // do not steal the active editor away
```

Comment:
@bhavyaus fyi, when an editor has focus, we open the walkthrough as inactive to not take the context away from the user at random.